projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -449,7 +471,10 @@ export class IgxComboComponent extends IgxComboBaseDirective implements AfterVie
             } else {
                 this._displayValue = this.createDisplayText(this.selection, args.oldSelection);
             }
-            this._onChangeCallback(args.newValue);
+            this.cdr.markForCheck();
+            if (emitEvent) {
```

Comment:
Not sure about this one - it shouldn't be tied to setting the value imo. Since we can set the value through the API and not emit an event, yet we still have to update the model.